serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/protobuf/ProtobufSerializer.java

```diff
@@ -121,7 +121,9 @@ public void validate(@NonNull Object object) {
         @Override
         public String load(@NotNull DescriptorProtos.FileDescriptorProto fileDescriptorProto) {
             final ProtoFileElement schemaElement = FileDescriptorUtils.fileDescriptorToProtoFile(fileDescriptorProto);
-            return schemaElement.toSchema();
+            String rawSchema = schemaElement.toSchema();
+            String stripped = "// Proto schema formatted by Wire, do not edit.\n// Source: \n\n";
```

Comment:
Shouldn't we be doing this in Apicurio library? That will ensure the behavior to be consistent everywhere. 

Also, if the generated schema doesn't have this string, it will truncate the actual schema text. Can we do a regex based truncation? Can you please also add a test?

Reply:
Yes ultimately we should change this in the Apicurio library, but this will require extra time to fix it and may delay the GA. 

As for now, if we are using the `FileDescriptorUtils`, the generated schema text will certainly have this text. I can add a test to verify that the schema definition generated is without this text. Also all tests are running to ensure that this removal won't have any issues.